docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -208,6 +208,181 @@ The content provided in your app must be easily understandable to all users. Tha
+Users should be able to resize text up to 200% without losing content or functionality.
+This guideline ensures that users with visual impairments, including those who rely on screen readers, can access and understand the content.
+
+:no_entry_sign: **Failure criteria**
+
+- Images of Text: It uses images of text without providing a text alternative, limiting access for users relying on screen readers.
+- Inadequate Contrast: The text in an image does not meet the required contrast ratio of 4.5:1, making it difficult for users with low vision to read.
+- Poor Text Alternatives: The contentDescription for an image of text is vague or does not convey the image’s meaning.
+- Cutoff Content: Text in images cannot be resized and becomes cut off or unreadable when enlarged.
+- Fixed Sizes: Images of text have fixed dimensions that do not adapt to user-defined text sizes, reducing accessibility.
+
+---
+### Text Spacing
+
+*This guideline covers point 1.4.12 Text Spacing - Level AA of the WCAG standard.*
+
+:white_check_mark: **Success criteria**
+
+Mobile apps must ensure that the following text spacing requirements are met to enhance readability:
+
+- Line Height: The line height (line spacing) must be at least 1.5 times the font size.
+- Paragraph Spacing: The spacing between paragraphs must be at least 2 times the font size.
+- Letter Spacing: The letter spacing (tracking) must be at least 0.12 times the font size.
+- Word Spacing: The word spacing must be at least 0.16 times the font size.
+- These spacing guidelines help ensure that users with low vision or reading difficulties can read text comfortably.
+
+These spacing guidelines help ensure that users with low vision or reading difficulties can read text comfortably.
+
+**Common Example:** Setting Text Spacing
+
+- **XML Example:** Use the following attributes in your `TextView` to ensure proper text spacing.
+
+```
+<!-- Example of setting text spacing in XML -->
+<TextView
+    android:id="@+id/spaced_text"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="This text is spaced according to WCAG guidelines."
+    android:textSize="16sp"
+    android:lineSpacingExtra="4dp"  <!-- Line height (1.5 times the font size) -->
+    android:letterSpacing="0.1"  <!-- Letter spacing (0.12 times font size) -->
+    android:paddingBottom="8dp" />  <!-- Paragraph spacing (2 times font size) -->
+```
+
+- **Jetpack Compose Example:** Use `lineHeight`, `letterSpacing`, and `padding` to achieve the desired spacing.
+
+```kotlin
+// Compose example with text spacing
+Text(
+    text = "This text is spaced according to WCAG guidelines.",
+    fontSize = 16.sp,
+    lineHeight = 24.sp,  // Line height (1.5 times font size)
+    letterSpacing = 0.1.em,  // Letter spacing (0.12 times font size)
+    modifier = Modifier.padding(bottom = 8.dp)  // Paragraph spacing (2 times font size)
+)
+```
```

Comment:
I like that you've added this section but I believe this criteria should be satisfied on the design level. 
Points that are listed as part of the success criteria should actually be guaranteed at the design level. Currently, it seems our responsibility is to make these adjustments in order to comply with the requirement. 

I am also okay with keeping the section to showcase how to implement these spacings but I would suggest we rephrase it a bit then. @Aksi0m what do you think?

Reply:
Makes sense, I agree 👍 
Let's emphasise that this is something that is handled on the design level. Leaving the impl example is fine for those cases when we need them.